packages/dfw/src/dfwcomponents/WorkOrderCell/WorkOrderCellNavigation.tsx

```diff
@@ -13,19 +13,20 @@ type WorkOrderCellNavigationProps = WorkOrderCellProps & {
 
 export const WorkOrderCellNavigation = ({
     title,
+    workOrderId,
+    workOrderType,
     maintenanceType,
     valueColor = "textTertiary",
     isHseCritical,
     isProductionCritical,
     showSymbols = true,
-    overwriteLabel,
     onPress,
     style,
     ...rest
 }: WorkOrderCellNavigationProps) => {
     const styles = useStyles(themeStyles);
 
-    const currentDate = moment();
+    const currentDate = useMemo(() => new Date(), []);
```

Comment:
Same here as for my comment on the WorkOrderCell in regards to passing this as a prop into the PropertyList component